packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -93,6 +92,9 @@ class PaginatedDataTable extends StatefulWidget {
   PaginatedDataTable({
     super.key,
     this.header,
+    this.headerBackgroundColor,
+    this.footerBackgroundColor,
+    this.footerStyle = const TextStyle(color: Color.fromARGB(255,0, 0, 0)), //default text style
```

Comment:
It looks like the default is supposed to be themeData.textTheme.bodySmall. This is a breaking change.